src/main/java/in/twizmwaz/cardinal/module/modules/startTimer/StartTimer.java

```diff
@@ -94,6 +92,12 @@ public void run() {
                 BossBar.delete();
             }
             time--;
+        } else if (GameHandler.getGameHandler().getMatch().getState().equals(MatchState.WAITING)) {
+            if (neededPlayers() > 0) {
+                BossBar.sendGlobalBossBar(waitingPlayerMessage(), 100F);
+            } else {
+                BossBar.delete();
```

Comment:
You'll only delete the boss bar if ```neededPlayers() <= 0```. I'd also add an automatic start if the ```neededPlayers() <= 0``` with a default ```neededPlayers()``` of 20 players.